code/examples/stderr/borrowck_4.stderr

```diff
@@ -12,10 +12,6 @@ error[E0505]: cannot move out of `foo` because it is borrowed
   |     borrow later used here
   |
   = note: this error originates in the macro `dbg` (in Nightly builds, run with -Z macro-backtrace for more info)
-help: consider cloning the value if the performance cost is acceptable
-  |
-7 |     mutate(&mut foo).clone();
-  |                     ++++++++
```

Comment:
why though?? x_x

this smels like a compiler bug

Reply:
might be fixed

Reply:
update: still a bug

Reply:
it's wrong to suggest to clone, since the value doesn't even implement clone. Newer nightlies suggest to derive clone, but that can't work either since you can't clone a &mut.....